Oneshot/README.md

```diff
@@ -1,8 +1,14 @@
 ## One-shot translation of lean 3 files
 
-This is a template which you can use to quickly play with the translation of snippets of lean 3 code. There are two inputs:
+This is a template which you can use to quickly play with the translation of snippets of lean 3 code. The procedure is:
 
-* `Oneshot/lean3-in/main.lean`: Put your lean code here.
-* `Oneshot/lean4-in/Oneshot.lean`: All alignments in `Mathlib` will be used, but you can add additional alignments here.
-
-To compile everything, (from the root of the repository) first run `make lean3-predata` beforehand, then run `make oneshot` after changing the inputs. The output is placed in `Outputs/src/oneshot/Oneshot/Main.lean`.
+1. Clone `https://github.com/leanprover-community/mathport/`
+```
+git clone https://github.com/leanprover-community/mathport/
+```
+2. get in the folder: `cd mathport`
+3. Run `make build` (if needed, run `sudo dnf install cmake gmp gmp-devel jq`)
```

Comment:
What's `dnf`? Wouldn't you normally be suggesting `apt` at this point? If I try running this on ubuntu I get a message telling me to install `dnf` using `apt`, so this doesn't seem to be a complete suggestion.

Reply:
Good question, I copy-pasted this from someone else on zulip. I'll fix this.